src/plugin.ts

```diff
@@ -82,14 +34,14 @@ export async function plugin(inputs: PluginInputs, env: Env) {
     } catch (e) {
       if (e instanceof CommanderError) {
         if (e.code !== "commander.unknownCommand") {
-          await context.logger.error(e.message);
+          context.logger.error(e.message);
```

Comment:
Perhaps I'm missing some large detail here but this used to write to our Supabase. I would expect this to be async.

Reply:
Yeah we removed all GitHub and Supabase integrations from the logger because it was dangerous to standardize it into plugins and have it be able to post directly to our db.

Also isn't it the case that we should only store to Supabase on `.fatal()` and only from inside either the kernel or core plugins?

I have suggested that the current logger implementation could be for non-core plugins and we have a core-logger which has the ability to post to our Supabase? Alternatively refactor the current logger but any way we refactor it opens us up to the issues we sought to resolve

Reply:
We can also post fatal to a dedicated telegram chat for us to triage. It might be simpler and more effective than a database specifically to triage errors.